crates/sui-types/src/unit_tests/utils.rs

```diff
@@ -157,58 +154,33 @@ pub fn mock_certified_checkpoint<'a>(
-        }
+    use fastcrypto_zkp::bn254::{utils::big_int_str_to_bytes, zk_login::ZkLoginInputs};
 
-        AUX_INPUTS.with(|a| a.clone())
-    }
+    use super::*;
 
-    fn get_public_inputs() -> PublicInputs {
+    fn get_inputs() -> ZkLoginInputs {
         thread_local! {
-            static PUBLIC_INPUTS: PublicInputs = PublicInputs::from_json(
-                "[\"17943667729617637440509062721605164279479613242834309109747499529639666176396\"]",
-            )
-            .unwrap();
+            static ZKLOGIN_INPUTS: ZkLoginInputs = ZkLoginInputs::from_json("{\"proof_points\":{\"pi_a\":[\"13051843614423432670927014216106415297302637061714876201698143724579485057433\",\"4502868464003469408525038047182455632737577658719986979336848385442638461715\",\"1\"],\"pi_b\":[[\"3766376969677702623227635810414794209012765519614143648451277080679068806865\",\"15274848924865252629859978623879023835428524488301134035498591572116063831502\"],[\"7946138017363758578225489333393267390091071305689481146893911497411367587561\",\"2088365154316551099212086738010689343716611426876233601699514012876088160321\"],[\"1\",\"0\"]],\"pi_c\":[\"5241274211994340733749437701839810664498991243102320350483342489924082173589\",\"15038895168244503564969393433273344359013838973322340723065298413943589759359\",\"1\"]},\"address_seed\":\"15454157267374145582481438333218897413377268576773635507925280300337575904853\",\"claims\":[{\"name\":\"iss\",\"value_base64\":\"wiaXNzIjoiaHR0cHM6Ly9pZC50d2l0Y2gudHYvb2F1dGgyIiw\",\"index_mod_4\":2},{\"name\":\"aud\",\"value_base64\":\"yJhdWQiOiJyczFiaDA2NWk5eWE0eWR2aWZpeGw0a3NzMHVocHQiLC\",\"index_mod_4\":1}],\"header_base64\":\"eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6IjEifQ\"}").unwrap().init().unwrap();
```

Comment:
based on the twitch test from fastcrypto https://github.com/MystenLabs/fastcrypto/blob/832925095f472bbc304795d8addbb74c59b79f01/fastcrypto-zkp/src/bn254/unit_tests/zk_login_tests.rs